app/src/main/java/org/oppia/android/app/walkthrough/ShowHideProgressBarAndHeaderTextview.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.walkthrough
+
+interface ShowHideProgressBarAndHeaderTextview {
```

Comment:
Please provide proper KDocs for this interface. Check other interfaces for an example.

Reply:
Done